dockers/base-cuda/Dockerfile

```diff
@@ -88,6 +88,8 @@ RUN \
     python ./requirements/pytorch/adjust-versions.py requirements/pytorch/base.txt ${PYTORCH_VERSION} && \
     python ./requirements/pytorch/adjust-versions.py requirements/pytorch/extra.txt ${PYTORCH_VERSION} && \
     python ./requirements/pytorch/adjust-versions.py requirements/pytorch/examples.txt ${PYTORCH_VERSION} && \
+    # remove colossalai from requirements since they are installed separately
```

Comment:
All the changes made to this `Dockerfile` also need to be made to `dockers/release/Dockerfile`, `build-pl` tests are failing because of that

Reply:
argggh.